src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 ## Unreleased
 
-## 1.4.0.beta.1
+## 1.4.0-beta.1
```

Comment:
Is the intention the version of this package aligned with OpenTelemetry SDK major.minor version?

Reply:
In other words, why is this `1.4.0-beta.1` instead of `1.3.1-beta.1`?

Reply:
minor version is incremented, because of adding new features, and not bug fixes (which would have been 1.3.1)

Reply:
Roger that, thanks!